Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -488,12 +488,21 @@ def audit_homepage
       end
     end
 
-    def audit_bottle_spec
+    def audit_bottle_disabled
       return unless formula.bottle_disabled?
       return if formula.bottle_disable_reason.valid?
       problem "Unrecognized bottle modifier"
     end
 
+    def audit_bottle_spec
+      return if formula.bottle_defined? || formula.bottle_disabled?
+      return if @new_formula
```

Comment:
Still probably want `bottle :unneeded` when needed on new formulae?

Reply:
And don't want `bottle :disabled` on new formulae either, I guess.

Reply:
I'm not sure I understand? New formulae won't have a bottle block so we can't check if they should be `bottle :unneeded`?

Reply:
Yeh I’m getting in a muddle. That’s a check that might make sense in ‘brew pull’ for maintainers (either pull a bottle on a new formula or have a bottle unneeded)

Reply:
But with bottle disabled: I think this won’t flag it on new formula in core? That seems fine but maybe weird.